app/controllers/home_controller.rb

```diff
@@ -3,6 +3,7 @@ class HomeController < ApplicationController
   def index
     if params[:q]
       @entries = Entry.search(params[:q])
+      @terms = Term.log_query(params[:q], @entries.count)
```

Comment:
Why do assign result of `Term.log_query` to the variable `@terms`?
It is not used anywhere and should be removed.